### PR TITLE
enhance NOENT error message with command you tried to use

### PR DESCRIPTION
**Pull request by EndangeredMassa**
When you don't have `java` installed, you will see:

```
Error: spawn ENOENT
    at errnoException (child_process.js:1001:11)
    at Process.ChildProcess._handle.onexit (child_process.js:792:34)
```

With this change you will see:

```
Error: Unable to find java
  at errnoException (child_process.js:1001:11)
  at Process.ChildProcess._handle.onexit (child_process.js:792:34)
```

---

Thanks to @jeloou for starting this work. This is an updated version of: https://github.com/groupon-testium/testium/pull/115

### CODE REVIEW

File: src/processes/server.coffee [hunk 1]
```diff
@@ -165,6 +167,15 @@ spawnServer = (logs, name, cmd, args, opts, cb) ->
         console.error err.stack
       throw error
 
+    child.baseUrl = "http://127.0.0.1:#{port}"
```
Comment (jkrems) [inline src/processes/server.coffee]: What was the reason for moving this?

Comment (EndangeredMassa) [inline src/processes/server.coffee]: Not in its current form, no.

**Comment by EndangeredMassa** [comment]
Simplified the diff.

File: lib/processes/server.js [hunk 2]
```diff
@@ -180,6 +184,13 @@ spawnServer = function(logs, name, cmd, args, opts, cb) {
       }
       throw error;
     });
+    child.baseUrl = "http://127.0.0.1:" + port;
```
Comment (jkrems) [inline lib/processes/server.js]: Compiled source missing?

Comment (EndangeredMassa) [inline lib/processes/server.js]: womp.

**Comment by EndangeredMassa** [comment]
Compiled.

**Comment by jkrems** [comment]
:shipit: when green.

**Comment by EndangeredMassa** [comment]
published in [2.5.0](https://github.com/groupon-testium/testium/releases/tag/v2.5.0)

**Pull request merged**